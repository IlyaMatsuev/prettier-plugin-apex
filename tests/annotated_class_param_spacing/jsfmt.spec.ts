runSpec(__dirname, ["apex"], {
  apexAnnotationsCamelCase: true,
  apexAnnotationsArgsSpacing: true,
});
