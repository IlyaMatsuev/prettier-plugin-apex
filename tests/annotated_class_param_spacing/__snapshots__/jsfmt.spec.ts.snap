// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Format apex: AnnotatedParamSpacing.cls: AnnotatedParamSpacing.cls 1`] = `
@isTest(SeeAllData=false) class AnnotatedParamSpacing
{}

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
@IsTest(SeeAllData = false)
class AnnotatedParamSpacing {
}

`;
